cboard-org/cboard-ai-builder
# Moving translation to fix production runtime

* Keeps `page.tsx` as a server component
* Creates an extra wrapper for Board page as a client component to keep `NextIntlClientProvider` to be imported from server component
* Removes `layout.tsx`


## Dev Notes
I don't have an obvious reason on why this bug is happening only in production mode (`yarn start`) besides thinking that this could be a behavior of NextIntl when injected directly from a client page.
I noticed that the `useMessage` hook can be used in a server component but the zustand state hook cannot, so in order to keep things to look like the other pages we built, I separated the server component importing the intl messages from the client component that calls the store.


## Steps for testing
1. `yarn install`
2. `yarn build`
3. `yarn start`
4. Manually test if the app works as expected
You can also run the docker container image by following the steps from [the wiki page](https://github.com/cboard-org/cboard-ai-builder/wiki/Running-with-Docker) 

Closes https://github.com/cboard-org/cboard-ai-builder/issues/113
